CONTRIBUTING.md

```diff
@@ -33,6 +33,20 @@ feel free to propose changes to this document in a pull request.
 * Feel free to open a Pull Request early so that a discussion can be had as changes are developed.
 * Include screenshots and animated gifs of your changes whenever possible.
 
+## Commit and Pull Request Semantics
+
+We use [Semantic Pull Requests](https://github.com/probot/semantic-pull-requests) to standardize our history
+
+### Commit Messages
+
+Prepend the type of change being made (refactor, fix, chore, feat, etc.) 
+Example: `fix: make header bold`
+
+### Pull Request Title
+
+Same as commit messages, prepend the type of change being made (refactor, fix, chore, feat, etc.) 
+Example: `docs: add linux setup instructions`
```

Comment:
I think it might be nice to leave it, just because it sums up conventional commits and will save a reader from having to dig into the conventional commits website and figure out how to apply it here, but happy to defer to your judgement

Reply:
We can leave it in. The [pull request docs for Electron itself](https://electronjs.org/docs/development/pull-requests#commit-message-guidelines) have examples.

Reply:
Well, we can get rid of the "Pull Request Titles" section - I've merged the two in my suggesions.